cluster/gce/util.sh

```diff
@@ -584,6 +584,26 @@ function kube-up {
   find-release-tars
   upload-server-tars
 
+  local running_in_terminal=false
+  if [ -t 1 ]; then
+    running_in_terminal=true
+  fi
+
+  if [[ ${running_in_terminal} == "true" || ${KUBE_UP_AUTOMATIC_CLEANUP} == "true" ]]; then 
```

Comment:
Other idea for logic here is to have a flag that turn the cleanup behavior on/off, always ask the user if we're running in a terminal and assume 'yes' as an answer if we're not. I think the logic will be simpler then. @zmerlynn ?